### PR TITLE
Bumps python to 2.7.18.

### PR DESCRIPTION
**Pull request by truthbk**
### What does this PR do?

Bumps python to 2.7.18.

### Motivation

Final python 2.7.18 release, ships a couple bugfixes.

### Additional Notes

This should be last the python update. Windows zipfiles have been built and uploaded.

### Describe your test plan

We should merge as soon as possible and issue will be apparent during the QA phase.

### CODE REVIEW

**Review by prognant — DISMISSED** [review]
LGTM

File: releasenotes/notes/python2-2_7_18-bump-b5aa063c56926564.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+upgrade:
```
Comment (olivielpeau) [inline releasenotes/notes/python2-2_7_18-bump-b5aa063c56926564.yaml:9]: I'd keep this under `enhancements`, I don't think it's worth an upgrade note (it's only a bugfix 2.7 bump after all):

```suggestion
enhancements:
```

(I think we should keep upgrade notes for significant changes that require user attention and may require user action upon upgrade)

File: releasenotes/notes/python2-2_7_18-bump-b5aa063c56926564.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  - |
+    Upgrading the python2 to the latest, and final, 2.7.18 release.
```
Comment (olivielpeau) [inline releasenotes/notes/python2-2_7_18-bump-b5aa063c56926564.yaml:11]: for consistency with other release notes :) :

```suggestion
    Upgrading the embedded Python 2 to the latest, and final, 2.7.18 release.
```

**Review by olivielpeau — DISMISSED** [review]
(just nits on the release note :) )

**Review by hush-hush — DISMISSED** [review]

**Review by KSerrania — APPROVED** [review]

**Pull request merged**